document_page/__manifest__.py

```diff
@@ -43,7 +32,5 @@
     'demo': [
         'demo/document_page.xml'
     ],
-    'installable': False,
```

Comment:
who recommends that? I usually ask people to remove keys when set to the default value